src/Login/Login.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useCallback, useContext, useState } from "react";
+import { Redirect } from "react-router";
+import firebase from "firebase/app";
+import PropTypes from "prop-types";
+import { AuthContext } from "../Auth";
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
Please use https://react-bootstrap.github.io/getting-started/introduction/, goes for all components :)